### ISSUE TITLE
every voucher has same hc_hash?

### ISSUE THREAD

**Issue by hnuyx** [fido-device-onboard/client-sdk-fidoiot#295]
https://github.com/fido-device-onboard/client-sdk-fidoiot/blob/d81474617b7fe0ef5492f0c28ed4508a820daf1d/lib/fdocred.c#L635
https://github.com/fido-device-onboard/client-sdk-fidoiot/blob/d81474617b7fe0ef5492f0c28ed4508a820daf1d/lib/prot/to2/msg63.c#L295

just a question: does every voucher have same hc-hash? is it designed for purpose？

**Comment by ben-krieger** [comment]
Yes, every voucher _entry_ has the same "Header Info" hash. I'm not exactly sure why, but see: https://fidoalliance.org/specs/FDO/FIDO-Device-Onboard-PS-v1.1-20220419/FIDO-Device-Onboard-PS-v1.1-20220419.html#OwnershipVoucher

```
;; ... each payload contains the hash of the previous entry
;; and the signature of the public key to verify the next signature
;; (or the Owner, in the last entry).
OVEntryPayload = [
    OVEHashPrevEntry: Hash,
    OVEHashHdrInfo:   Hash,  ;; hash[GUID||DeviceInfo] in header
    OVEExtra:         null / bstr .cbor OVEExtraInfo
    OVEPubKey:        PublicKey
]
```

Since GUID||DeviceInfo does not change, the hash does not change.
